### DISCUSSION TITLE
Can CVE severity change?

### DISCUSSION THREAD

**Discussion by jagnjilo1234** [aquasecurity/trivy#5073] [Q&A] [answered]
### Question

Hello,
Our company is currently using Harbor as their docker repository, and Harbor uses Trivy to check for vulnerabilities. We do not allow images that contain CVEs with severity High or Critical to be pulled. However, sometimes the effort to fix a vulnerability is too high so we whitelist it. We have now, on multiple occasions had a problem where we whitelist all high and critical CVEs, the images can be pulled without any issues, but after a couple of days the images can not be pulled and a new high CVE arises without any new scans happening. Our theory is that the severity changes from Medium to High, so my question is - is it possible that a CVE severity can change?

### Target

None

### Scanner

Vulnerability

### Output Format

None

### Mode

None

**Comment by knqyf263** [comment, marked as answer]
Yes. For example, NVD has the change history at the bottom. You can see if the severity was changed.
